app/controllers/measurement_sessions_controller.rb

```diff
@@ -21,23 +21,11 @@ class MeasurementSessionsController < ApplicationController
 
   def show
     session = Session.find_by_url_token(params[:url_token]) or raise NotFound
-    lat = session.streams.first.min_latitude.to_f
-    lng = session.streams.first.min_longitude.to_f
-    stream = session.streams.first
 
-    map = { zoom:16, lat:lat, lng:lng }
     selected_session_ids = [session.id]
-    data = {
-      heat: { highest: stream.threshold_very_high,
-              high: stream.threshold_high,
-              mid: stream.threshold_medium,
-              low: stream.threshold_low,
-              lowest: stream.threshold_very_low },
-      usernames: session.user.username,
-      sensorId: stream.sensor_id
-    }
+    data = { sensorId: "Temperature-airbeam2-f (F)" }
```

Comment:
There is a separate ticket that will adjust this endpoint to accept the selected stream form the mobile app.